src/client/src/ui/blotter/views/blotterView.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { Table, Column, Cell } from 'fixed-data-table';
 import { DateCell, NotionalCell } from './';
-import Dimensions from 'react-dimensions';
+import Dimensions from './../../../system/dimensions';
```

Comment:
This should look like this once moved correctly:
import { Dimensions } from '...';